src/Lib.hs

```diff
@@ -3,35 +3,20 @@ module Lib
     ) where
 
 
-import           Commands           (add, ls)
+import           Commands.Add       (add)
+import           Commands.Help      (help)
+import           Commands.List      (ls)
+import           Commands.NotFound  (notFound)
 import           System.Environment (getArgs)
 
 
-cmds :: String
-cmds =
-    "usage: timetrack <command>\n\n" ++
-        "  add  Add entry\n" ++
-        "  ls   List entries"
-
-
-cmdNotFound :: String -> [String] -> IO ()
-cmdNotFound cmd _ = do
-    putStrLn $ "No such command: " ++ cmd
-    putStrLn cmds
-
-
-help :: [String] -> IO ()
-help _ =
-    putStrLn cmds
-
-
 dispatch :: String -> [String] -> IO ()
 dispatch "add"    = add
 dispatch "list"   = ls
 dispatch "ls"     = ls
 dispatch "-h"     = help
 dispatch "--help" = help
-dispatch cmd      = cmdNotFound cmd
+dispatch cmd      = notFound cmd
```

Comment:
Looks like the `timeTrack` function that calls this currently errors if no args are given. Should that also display a help message?

Reply:
yes, it should!